lib/miq_automation_engine/service_models/miq_ae_service_orchestration_stack.rb

```diff
@@ -11,6 +11,7 @@ class MiqAeServiceOrchestrationStack < MiqAeServiceModelBase
     expose :raw_delete_stack
     expose :raw_update_stack
     expose :raw_exists?
+    expose :refresh, :method => :refresh_ems
```

Comment:
@Ladas 
Can we call the method refresh_ems instead of refresh

```
expose :refresh_ems
```

Reply:
/we use the same naming in vm_or_template and other I think, so lets keep it consistent, or rename all of them?

Reply:
hm I think we can't rename them all really, we must be backwards compatible for Automate scripts, so vm.refresh must stay